### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by John-S4**
Small updates and clarifications regarding branding, informality of community discussions, and committee change policy.

### CODE REVIEW

**Review by evrifaessa — APPROVED** [review]
Well-done and clear.

**Review by jaapmarcus — APPROVED** [review]

**Comment by 4yx** [comment]
> Third-party extensions to FOSSBilling are also welcomed, it would be appreciated if these were discussed and informed by the community
what does this mean, exactly? that it would be nice if the community was informed about them, or if the community informs (the governing team?) about them? "by" feels a little off..

File: GOVERNANCE.md [hunk 1]
```diff
@@ -84,6 +86,6 @@ In general, if nobody explicitly opposes a proposal, it is recognised as having
 
 ## Changes to this governance policy
 
-Any community member is free to propose changes to this governance policy by submitting a proposed change to this document. The steering committee will discuss any proposed changes and aim to reject or accept the changes by consensus.
+Any community member is free to propose changes to this governance policy by submitting a proposed change to this document, via GH pull request. The steering committee will discuss any proposed changes and aim to reject or accept the changes by consensus.
```
Comment (BelleNottelling) [inline GOVERNANCE.md:89]: Let's actually write out the word GitHub

File: GOVERNANCE.md [hunk 2]
```diff
@@ -17,10 +17,12 @@ __TLDR; Project decisions are made by a consensus of the projects steering commi
 
 The FOSSBilling community is made up of individuals, companies, or groups who use and/or contribute to the FOSSBilling software.
 
-Anybody is free to use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements.
+Anybody is free to use the core FOSSBilling software within the terms of the [license](https://github.com/FOSSBilling/FOSSBilling/blob/main/LICENSE). There are not and never will be any fees for usage, and no special requirements. 
 
 We really appreciate user contributions but if you just want to download, install, and go, then you have the complete freedom to do that.
 
+Third-party extensions to FOSSBilling are also welcomed, it would be appreciated if these were discussed and informed by the community, but this is not mandatory. In these cases please do be aware of the restrictions attached to usage of the [FOSSBilling name and branding](https://github.com/FOSSBilling/branding#readme).
```
Comment (BelleNottelling) [inline GOVERNANCE.md:24]: And this part also just feels a bit strange to me, it's worded a bit confusingly and I'm not entirely sure what purpose it serves

**Comment by John-S4** [comment]
> 



> what does this mean, exactly? that it would be nice if the community was informed about them, or if the community (the developers of the 3rd party extensions, as members of the community) informs (the governing team?) about them? "by" feels a little off...
> Would "shared and discussed with the community" be close to what you mean, or have I interpreted it wrongly?

I worded it badly, now updated, let me know if it is now clearer.

File: GOVERNANCE.md (hunk 1 above)
Comment (John-S4) [reply GOVERNANCE.md:89]: Done

File: GOVERNANCE.md (hunk 2 above)
Comment (John-S4) [reply GOVERNANCE.md]: Reworded - the purpose was essentially drawing attention to branding restrictions for third-party modules/integrations.

File: GOVERNANCE.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ Anybody is free to use the core FOSSBilling software within the terms of the [li
 
 We really appreciate user contributions but if you just want to download, install, and go, then you have the complete freedom to do that.
 
-Third-party extensions to FOSSBilling are also welcomed, it would be appreciated if these were discussed and informed by the community, but this is not mandatory. In these cases please do be aware of the restrictions attached to usage of the [FOSSBilling name and branding](https://github.com/FOSSBilling/branding#readme). 
+Third-party modules and integrations for FOSSBilling are welcomed. We would apprciate if your third-party work was announced to the community so that it can be added to the list of known working extensions, however this is not mandatory. In all cases, please be aware of the restrictions attached to usage of the [FOSSBilling name and branding](https://github.com/FOSSBilling/branding#readme), and approach us for permission if you wish to use the branding in any way other than explicitly listed.
```
Comment (4yx) [inline GOVERNANCE.md:24]: Nice, seems very clear! Just a small typo on "appreciate" :D

**Review by 4yx — APPROVED** [review]

File: GOVERNANCE.md (hunk 3 above)
Comment (John-S4) [reply GOVERNANCE.md]: Fixed, I appreciate you noticing :)

File: GOVERNANCE.md [hunk 4]
```diff
@@ -84,6 +86,6 @@ In general, if nobody explicitly opposes a proposal, it is recognised as having
 
 ## Changes to this governance policy
 
-Any community member is free to propose changes to this governance policy by submitting a proposed change to this document. The steering committee will discuss any proposed changes and aim to reject or accept the changes by consensus.
+Any community member is free to propose changes to this governance policy by submitting a proposed change to this document, via GitHub pull request. The steering committee will discuss any proposed changes and aim to reject or accept the changes by consensus.
```
Comment (BelleNottelling) [inline GOVERNANCE.md:89]: Shouldn't it be "via a GitHub pull request"?

**Comment by John-S4** [comment]
Please nobody merge until other SC members have had a chance to review.

File: GOVERNANCE.md (hunk 4 above)
Comment (John-S4) [reply GOVERNANCE.md:89]: I don't completely agree, but it isn't important and I'm happy to go with the majority so I will change it.

**Comment by John-S4** [comment]
Just as a heads up, I am going to merge this at the end of the afternoon if there are no further comments.

**Review by BelleNottelling — APPROVED** [review]

**Pull request merged**